source/community/about/community-guidelines.html.md

```diff
@@ -8,89 +8,43 @@ wiki_revision_count: 11
 wiki_last_updated: 2015-05-20
 ---
 
-# Community guidelines
+# Community Code of Conduct
 
-We want to keep the oVirt Community a great place to participate, but we need your help to keep it that way. While we have specific guidelines for various tools, in general, you should:
+We want to keep the oVirt Community a great place to participate, but we need your help to keep it that way. The oVirt community is made up of a diverse mix of individuals using and contributing to all aspects of the projects all over the world, and we want to make sure that the community is a safe and friendly place for everyone. 
```

Comment:
"contributing to all aspects of the projects" sounds a bit odd, as if something is missing there. How about "contributing to all aspects of the various projects that make up oVirt", or something like that?